data/mods/ssb/random-teams.ts

```diff
@@ -578,6 +584,13 @@ export class RandomStaffBrosTeams extends RandomTeams {
 			// Any set specific tweaks occur here.
 			if (set.name === 'quadrophenic') set.moves[this.random(2) + 1] = 'Conversion';
 
+			if (set.name === 'Marshmallon') {
+				if (set.moves[1] === set.moves[2]) {
+					const moveChoices = ['Flare Blitz', 'Wood Hammer', 'Head Smash'];
+					set.moves[2] = moveChoices.filter((move: string) => move !== set.moves[1])[this.random(2)];
+				}
+			}
```

Comment:
this is not going to fix the problem you are trying to solve. Just randomly generate all 3 moves and replace one with head charge if it isnt there

Reply:
it should replace the third move if it is same as the second

Reply:
true, but if you rng say, Flare Blitz, Head Charge, Flare Blitz, then that filter wont catch it.

Reply:
hmmm okay